msteams-platform/bots/how-to/bot-configuration-experience.md

```diff
@@ -54,95 +65,673 @@ You must configure the `fetchTask` property under `bots.configuration` object in
+For more information, see [app manifest schema](../../resources/schema/manifest-schema.md#botsconfiguration).
 
 ### Configure your bot
 
-When a user installs the bot in a team or group chat scope, the `fetchTask` property in the app manifest file initiates `config/fetch` defined in the `teamsBot.js` file. The bot responds with an Adaptive Card and the user provides relevant information in the Adaptive Card and selects **Submit**. After the user selects **Submit**, a `config/submit` is returned to the bot and the bot configuration is complete.
+When a user installs the bot in a team or group chat scope, the `fetchTask` property in the app manifest file initiates `config/fetch` or `config/submit`. The bot responds with an Adaptive Card, when the user provides relevant information in the Adaptive Card and selects Submit, `config/submit` or `config/continue` is returned to the bot and the bot configuration is complete.
+
+You can use the following methods to enable configuration settings for a bot:
+
+1. [Non-Teams method](#non-teams-method).
+
+1. [Teams-specific method](#teams-specific-method).
 
-You can use `config/fetch` and `config/submit` properties in the `teamsBot.js` file to enable the bot configuration experience in a team or group chat.
+#### Non-Teams method
 
-* Invoke `config/fetch`: You must invoke `config/fetch` task to start the bot configuration flow and define the bot response as required.
+The Non-Teams method simplifies task execution using `OnInvokeActivity` and `OnInvokeActivityAsync`, removing the need for Teams integration:
 
-  In the following example, when the bot receives `config/fetch` invoke activity, the bot responds with an Adaptive Card:
+1. `OnInvokeActivity`: The `OnInvokeActivity` method is a fundamental component of the Bot Framework SDK. It enables users to handle incoming activities that aren't recognized by any other method in the bot's logic. This method provides flexibility for handling a wide range of activities, including user interactions, within the bot's conversation flow.
+
+   # [JavaScript](#tab/JS1)
+
+    The `onInvokeActivity` function simplifies the handling of invoke activities. For `config/fetch` activities, it allows continuation of the configuration with an Adaptive Card or signals completion with a message. For `config/submit` activities, it either proceeds with further configuration through another Adaptive Card or ends the process with a completion message. The Adaptive Cards, `adaptiveCardForContinue` and `adaptiveCardForSubmit` are designed with specific layouts and interactive elements.
 
     ```javascript
-    if (context._activity.name == "config/fetch") {
-      const adaptiveCard = CardFactory.adaptiveCard(this.adaptiveCardForDynamicSearch());
-      try {
+    async onInvokeActivity(context) {
+ 
+    if (context._activity.name == "config/fetch"){
+      
+      /*
+      Option 1: You can add a "config/auth" response as below code
```

Comment:
This "config/auth" response is mentioned for the **first** time here, in this code sample!